app/uploaders/profile_photo_attachment_uploader.rb

```diff
@@ -7,7 +7,9 @@ class ProfilePhotoAttachmentUploader < CarrierWave::Uploader::Base
   include AsyncProcessing
   include LogMetrics
 
-  MAX_FILE_SIZE = 10.megabytes
+  def self.max_file_size
```

Comment:
What is your reasoning for changing this to a class method vs a constant? I was thinking that if something is static, a constant is preferred, but since you’re referencing it in the ValidateFileSize class and it's publicly exposed, maybe the class method is the better choice. 

Sorry, probably nitpicky, lots of coffee this am :)

Reply:
If I'm going to return a http 413 I need a way to trigger the "file too large" error.  I'd rather override the `max_file_size` than add a >50 mb file for a spec (50 is the max elsewhere).